src/components/Invoices/InvoicesTable/InvoicesTable.tsx

```diff
@@ -38,7 +39,7 @@ enum InvoiceStatusFilter {
   Voided = 'Voided',
 }
 
-type InvoiceStatusOption = {
+export type InvoiceStatusOption = {
   label: string
   value: InvoiceStatusFilter
 }
```

Comment:
Minor nit: This like this might belong in `InvoiceProvider` or some shared types file now.